doc/nrf/../fota: documented auto confirmation of direct-XIP image (#17419)
nrfconnect/sdk-nrf: doc/nrf/app_dev/device_guides/nrf52/fota_update.rst

    west build -b *board_target* -- -DSB_CONFIG_BOOTLOADER_MCUBOOT=y -DSB_CONFIG_MCUBOOT_MODE_DIRECT_XIP_WITH_REVERT=y -DCONFIG_NCS_SAMPLE_MCUMGR_BT_OTA_DFU=y

.. note::

>    When building the application with MCUboot in DirectXIP mode with revert mechanism support, the signed image intended for flashing is automatically marked as confirmed.

Reviewer: ```suggestion
   When building the application with MCUboot in direct-xip mode with revert mechanism support, the signed image intended for flashing is automatically marked as confirmed.
```

>    When building the application for  with MCUboot in direct-xip mode and the revert mechanism support, the signed image intended to be flashed is marked confirmed.
>    If the application would be built and flashed without this option, it will fail to boot.

Reviewer: ```suggestion
   When building the application with MCUboot in the direct-xip mode with revert mechanism support, the signed image intended for flashing is automatically marked as confirmed.
   Without this configuration, the application will fail to boot.
```

Reviewer 2: *directXIP

Reviewer: fixed
